packages/appsync-emulator-serverless/dynamodbSource.js

```diff
@@ -380,17 +380,17 @@ const batchDeleteItem = async (db, dynamodbTables, { tables }) => {
 const resolve = async (dynamodb, table, dynamodbTables, payload) => {
   switch (payload.operation) {
     case 'GetItem':
-      return getItem(dynamodb, table, payload);
+      return getItem(dynamodb, dynamodbTables[table], payload);
```

Comment:
Instead of sprinkling logic all over to handle dynamodbTable mapping I realized I could do it all here! I think this is a win